tests/ci/azure_artifact_feed.yaml

```diff
@@ -0,0 +1,91 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+# To install recommenders artifact:
+# $ pip install artifacts-keyring --pre
+# Choose the version to install based on your environment:
+# $ pip install reco-utils --index-url=https://pkgs.dev.azure.com/best-practices/recommenders/_packaging/recommenders/pypi/simple/
+# $ pip install reco-utils-gpu --index-url=https://pkgs.dev.azure.com/best-practices/recommenders/_packaging/recommenders/pypi/simple/
+# $ pip install reco-utils-pyspark --index-url=https://pkgs.dev.azure.com/best-practices/recommenders/_packaging/recommenders/pypi/simple/
+
+
+# NOTE: 
+# This yaml file is used to generate ADO artifacts for Recommenders library.
+# We use the same yaml file to generate the library in different environments: CPU, GPU and PySpark
+# When setting the pipeline, we need to define the variables env_name, env_flag and library_name
+# To add the variables, go to the pipeline, press edit and then variables.
+# An example of the variables we use in the GPU environment:
+# env_name=artifact_reco_gpu
+# env_flag=--gpu
+# library_name=reco_utils_gpu
+
+# The global variables are defined in ADO/Pipelines/Library
+# https://dev.azure.com/best-practices/recommenders/_library?itemType=VariableGroups
+variables:
+- group: LinuxAgentPool
+
+# Pull request against these branches will trigger this build

```

Comment:
As we discussed on call,
We need not trigger on PR. If I raise a PR to master accidentally new package will be uploaded.

Reply:
great catch, fixed